Downeastrussell/Spend-It
# Manage My Locations

CRUD--edit, create working for MyLocations
VIEWS--Home(details), Manage My Locations(edit,details,delete)

to test:
-login
-navigate to "Manage My Location" & create a new location
--select "edit" on newly created Location, change fields, and click save
--changes should be reflected under "Home" and "Manage My Locations" index views
